src/ensembl/src/content/app/contact-us/ContactUs.tsx

```diff
@@ -0,0 +1,63 @@
+/**
```

Comment:
But ContactUs is not an app! Not in the sense that GenomeBrowser or EntityViewer are. It's a shared component.

Reply:
Also, the name of this thing should not be `ContactUs`. According to Andrea, this layout:

![image](https://user-images.githubusercontent.com/6834224/127141956-7357abca-6fc0-4d7b-8a0a-056e9d8b7853.png)

is also going to be used to show messages to the user, as can be seen from the header of this panel:

![image](https://user-images.githubusercontent.com/6834224/127142175-e6358695-255b-4bb8-b0b0-2d5b3f71c292.png)

`ContactUs` is just one of the components that's going to be placed in the panel.

Reply:
I've renamed it to `CommunicationPanel`. Please let me know if you can think of a better name.

Reply:
The directory structure currently looks like this:

<img src="https://user-images.githubusercontent.com/6834224/127155237-7e7a5b5c-42d1-4166-a432-737a262f1d5c.png" width="300"/>

one possibility is to regroup it into something like this:

<img src="https://user-images.githubusercontent.com/6834224/127155391-4bdb2843-9c95-4e09-88c4-fecff5045fd9.png" width="280"/>

One benefit of that is that closely related components will be co-located (we aren't going to use the communication button component without it calling the communication panel component).